examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -14,29 +14,40 @@
   },
```

Comment:
<p>This should be "find out if there have been divergent transitions and how many"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>